packages/otelbin-validation-image/src/lambda-wrapper.ts

```diff
@@ -0,0 +1,85 @@
+const { NodeTracerConfig, NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const {
+	BatchSpanProcessor,
+	ConsoleSpanExporter,
+	SDKRegistrationConfig,
+	SimpleSpanProcessor
+} = require("@opentelemetry/sdk-trace-base");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { awsLambdaDetector } = require("@opentelemetry/resource-detector-aws");
+const { detectResourcesSync, envDetector, processDetector } = require("@opentelemetry/resources");
+const { AwsInstrumentation } = require("@opentelemetry/instrumentation-aws-sdk");
+const {
+	AwsLambdaInstrumentation,
+} = require("@opentelemetry/instrumentation-aws-lambda");
+const { diag, DiagConsoleLogger, DiagLogLevel } = require("@opentelemetry/api");
+const { getEnv } = require("@opentelemetry/core");
+const { OTLPTraceExporter } = require("@opentelemetry/exporter-trace-otlp-proto");
+const { MeterProvider, MeterProviderOptions } = require("@opentelemetry/sdk-metrics");
+
+function defaultConfigureInstrumentations() {
```

Comment:
For the most part, this is a copy of the official Node.js lambda layer's wrapper file. Unfortunately reusing as dependency is not possible, because it is not published as an NPM package.

Reply:
_suggestion:_ How about adding a comment to that extent as a maintenance note? So we know which code we vendored-in here an why.